incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
```

Comment:
Are you intending for this to be automatically built and pushed by the CI? Currently, it's not clear to me how this image ends up in `gcr.io/kubernetes-charts-ci/jenkins-master-k8s`.